quickstart-node/package.json

```
@@ -0,0 +1,24 @@
+{
+  "name": "quickstart-node",
+  "version": "1.0.0",
+  "description": "",
+  "scripts": {
+    "build": "npx tsc",
```

```suggestion
    "build": "tsc",
```

Use typescript as a dev dependency